README.md

```diff
@@ -130,40 +130,42 @@ Otherwise, you can simply import along with the backend itself (as shown above).
 <a name="Options"></a>
 ### Options
 
-- [`tags`](#tagsOption)
-- [`suggestions`](#suggestionsOption)
-- [`delimiters`](#delimiters)
-- [`placeholder`](#placeholderOption)
-- [`labelField`](#labelFieldOption)
-- [`handleAddition`](#handleAdditionOption)
-- [`handleDelete`](#handleDeleteOption)
-- [`handleDrag`](#handleDragOption)
-- [`handleFilterSuggestions`](#handleFilterSuggestions)
-- [`handleTagClick`](#handleTagClickOption)
-- [`autofocus`](#autofocus)
-- [`allowDeleteFromEmptyInput`](#allowDeleteFromEmptyInput)
-- [`handleInputChange`](#handleInputChange)
-- [`handleInputFocus`](#handleInputFocus)
-- [`handleInputBlur`](#handleInputBlur)
-- [`minQueryLength`](#minQueryLength)
-- [`removeComponent`](#removeComponent)
-- [`autocomplete`](#autocomplete)
-- [`readOnly`](#readOnly)
-- [`name`](#nameOption)
-- [`id`](#idOption)
-- [`maxLength`](#maxLength)
-- [`inline`](#inline)
+Option | Default
```

Comment:
can you also add `Type` as 2nd column where type of each prop is mentioned ?
Also `description` as 4th column where its mentioned whether the prop is required and the usage of the prop in `1-2` lines ?